src/Quasar/Advanced/QuasarAF.purs

```diff
@@ -95,7 +96,7 @@ readQueryEJson
   → Maybe Pagination
   → QuasarAFCE (Array EJson)
 readQueryEJson path sql vars pagination =
-  readQuery Precise path sql vars pagination <#> resultsAsEJson
+  readQuery Precise path sql vars pagination <#> resultsAsEJson'
```

Comment:
Have you considered not changing name of old `resultsAsEJson`? 

in SD we use the old `resultsAsEJson` , so if we do not changed the name we would have less diffs here and in SD too, and it would be less braking.

Reply:
If I understand correctly, this was a new `resultsAsEJson'` for the newly made cache handling.

`resultsAsEJson` still exists.

Reply:
Yes you still need both, but the one which was there before, with name `resultsAsEJson` has same type as `resultsAsEJson'`. because of this you had updated all usages of old `resultsAsEJson` with new `resultsAsEJson'`, like this line here. So to make this change less braking we should swap names.

You can do this easily by doing 3 replaces in whole source code:

1. replace `resultsAsEJson'` with `SOME_UNIQUE_TEMP_STRING`
2. replace `resultsAsEJson` with `resultsAsEJson'`
3. replace `SOME_UNIQUE_TEMP_STRING` with `resultsAsEJson`

as a result names will be swapped :P